config/default.yml

```diff
@@ -174,6 +174,18 @@ Bundler/GemComment:
   IgnoredGems: []
   OnlyFor: []
 
+Bundler/GemVersionDeclaration:
+  Description: 'Requires or prohibits gem version declarations.'
+  Enabled: false
+  VersionAdded: '<<next>>'
+  EnforcedStyle: 'required'
+  SupportedStyles:
+    - 'required'
+    - 'prohibited'
+  Include:
+    - '**/Gemfile'
+  IgnoredGems: []
```

Comment:
Lately I've favoured more names like "AllowedGems" instead of "IgnoredGems". Sadly, our use of both is a bit inconsistent.

Reply:
+1 for `AllowedGems`.